### PR TITLE
docs: add example for mongoose connection injection

### PR DESCRIPTION
**Pull request by quen2404**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Currently, there is no example about injecting mongoose connection with  `@nestjs/mongoose`.

Issue Number: N/A

## What is the new behavior?
A new section called `Connection Injection` is added in `Techniques / Mongo` page.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: src/app/homepage/pages/techniques/mongo/mongo.component.html [hunk 1]
```diff
@@ -57,6 +57,12 @@ <h4>Model injection</h4>
   </span>
   <pre [class.hide]="catsServiceT.isJsActive"><code class="language-typescript">{{ catsService }}</code></pre>
   <pre [class.hide]="!catsServiceT.isJsActive"><code class="language-typescript">{{ catsServiceJs }}</code></pre>
+  <h4>Connection Injection</h4>
+  <p>
+    Maybe sometime you will need the native <a href="https://mongoosejs.com/docs/api.html#Connection">Mongoose Connection</a>
+    (for an integration with an external library, or to use the native API). In such case, use the <code>@InjectConnection</code> decorator:
```
Comment (LukvonStrom) [inline src/app/homepage/pages/techniques/mongo/mongo.component.html]: Please add a `s` after `case`

**Comment by kamilmysliwiec** [comment]
Thanks for your contribution! However, every page has been moved to the markdown syntax now (`content` directory) which makes this PR outdated. Could you please update it?

**Comment by quen2404** [comment]
Hi @kamilmysliwiec ! OMW to change that including @LukvonStrom fix :wink:

**Comment by quen2404** [comment]
DONE @kamilmysliwiec :)

**Comment by kamilmysliwiec** [comment]
Could you review @johnbiundo? :)

**Pull request merged**